addons/mail/static/src/messaging/component/discuss_mobile_mailbox_selection/discuss_mobile_mailbox_selection.js

```diff
@@ -1,27 +1,27 @@
-odoo.define('mail.component.DiscussMobileMailboxSelection', function (require) {
+odoo.define('mail.messaging.component.DiscussMobileMailboxSelection', function (require) {
 'use strict';
 
-const useStore = require('mail.hooks.useStore');
+const useStore = require('mail.messaging.component_hook.useStore');
 
 const { Component } = owl;
 const { useGetters } = owl.hooks;
 
-class MobileMailboxSelection extends Component {
+class DiscussMobileMailboxSelection extends Component {
 
     /**
      * @override
-     * @param {...any} args
      */
     constructor(...args) {
         super(...args);
         this.storeGetters = useGetters();
         this.storeProps = useStore(() => {
             return {
-                pinnedMailboxList: this.storeGetters.pinnedMailboxList(),
+                allOrderedAndPinnedMailboxes:
+                    this.storeGetters.allOrderedAndPinnedMailboxes(),
```

Comment:
I would prefer to create a variable instead of going to new line in that case (both branch).